test/packages/EABase/include/Common/EABase/eahave.h

```diff
@@ -559,7 +559,7 @@
 #endif
 
 #if !defined(EA_HAVE_ISNAN) && !defined(EA_NO_HAVE_ISNAN)
-	#if defined(EA_PLATFORM_MICROSOFT) && !defined(EA_COMPILER_CLANG)
+	#if defined(EA_PLATFORM_MICROSOFT) && !defined(EA_PLATFORM_MINGW)
```

Comment:
MinGW isn't the only way clang can run on Windows.  We'd need MinGW have to be a seperate case in this block... similar to how Cygwin is handled.